### DISCUSSION TITLE
New Features

### DISCUSSION THREAD

**Discussion by fenneishi** [fenneishi/Fooocus-Control#11] [Show and tell]
<div align="center">

# 🔥🔥🔥**New Feature Coming: mask inpaint **🔥🔥🔥

#### Go to [Install](https://github.com/fenneishi/Fooocus-ControlNet-SDXL#%EF%B8%8Finstall%EF%B8%8F)🛠️

</div>

---

![image](https://github.com/fenneishi/Fooocus-ControlNet-SDXL/assets/33345538/88182608-0283-427f-b4d3-98f0bc481aa3)

**Comment by XiaoCC** [comment]
太棒了 非常需要，什么时候能合并入主分支呢

**Comment by fenneishi** [comment]
It's ready to use. Download it directly
